packages/pluggableWidgets/video-player-native/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "video-player-native",
   "widgetName": "VideoPlayer",
-  "version": "4.0.0",
+  "version": "4.0.1",
```

Comment:
This change doesn't sound like a patch version increase, I would make it a major one.